### ISSUE TITLE
Introduction to Supervised Learning

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/DS-ScriptsNook#82]
Hey DCP participants, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Introduction to Supervised Learning. Tutorial to Supervised Learning and how it's applied using sample code.

"Project Title" should be the "Issue Title" mentioned here.

If you like to work on this issue, follow the given guidelines for code as well as for the README. You have to be as elaborative as possible. Make sure you also maintain the project structure provided. Just comment and get it assigned. Issues are alloted on FCFS basis.

The main intention of this repo is to educate beginners with all the concepts related to datascience so that they could feel confident to apply these things learnt in practical exposure.

Enjoy your open source journey ahead. 😎

**Comment by ashish-reddy-20-08** [comment]
@prathimacode-hub , can you please assign me this

**Comment by ramya-neelakantan** [comment]
Hi! I'd like to work on this for Hacktoberfest 21. Can you please assign it to me? 
I plan to write detailed information on how supervised learning is applied, along with sample codes and examples.

**Comment by prathimacode-hub** [comment]
Issue is assigned to @ramya-neelakantan

**Comment by prathimacode-hub** [comment]
@ashish-reddy-20-08, can you choose some other issue. Also while asking your assignment mention the approach and through what open source program are you taking up this issue.

**Comment by prathimacode-hub** [comment]
Yes, its all mentioned in contributing guidelines and while create PR use the PR template given. Along with that, you can find the project structure that needs to be followed in README.md file of thie repository. @ramya-neelakantan
